app/src/main/java/org/greenstand/android/TreeTracker/activities/ImageCaptureActivity.kt

```diff
@@ -88,6 +84,7 @@ class ImageCaptureActivity : AppCompatActivity() {
                     }
 
                     override fun onImageSaved(file: File) {
+                        ImageUtils.resizedImage(file.absolutePath)
```

Comment:
@arunbakt This is the only line with any real changes. I resize the image after it's taken using existing resize logic.

Reply:
Got that, thank you!